README.md

```diff
@@ -0,0 +1,59 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo List app
+
+> A simple todo list web app build with Javascript.
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
```

Comment:
- [ ] Good job so far 👏 Please add procedures to clone and run your repo locally. This helps your fellow developers to know how to use your project 😃 👍